lxd/api.go

```diff
@@ -165,6 +165,7 @@ func restServer(d *Daemon) *http.Server {
 	})
 
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		countStartedRequest(r)
```

Comment:
For example im not sure we should be tracking redirections to /ui

Reply:
My reasoning for including this was only because the `/` is mentioned in the [main API spec](https://documentation.ubuntu.com/lxd/en/latest/api/#/). But indeed it makes sense to not include it, I will just add a mention to this in the metrics docs to avoid confusion.